roles/common/tasks/subscription_tests.yml

```diff
@@ -0,0 +1,8 @@
+---
+
+- name: Verify subscription
+  ansible.builtin.shell:
+    cmd: |
+      oc get subscriptions -n "{{ subscription_nspace }}" "{{ item }}" 
```

Comment:
IMHO, this should have a similar format to the endpoint tests, so that a single loop can cover multiple namespaces.